scripts/js-analyze.js

```diff
@@ -1446,8 +1470,31 @@ let Analyzer = {
       break;
     }
   },
+
+  maybeLinkifyLiteral(expr) {
+    if (typeof expr.value !== "string") {
+      return;
+    }
+
+    if (!expr.value.startsWith("chrome://") && !expr.value.startsWith("resource://")) {
+      return;
+    }
+
+    ensureURLMap();
+
+    if (!(expr.value in urlMap)) {
+      return;
+    }
+    targetPath = urlMap[expr.value];
+
+    this.source(expr.loc, "\"" + expr.value + "\"", "file,use", "file " + targetPath, "FILE_" + atEscape(targetPath));
```

Comment:
It would be great to emit a target record here, too!  I'm not sure if you noticed, but there are `(file symbol)` spans to the right of the bread crumbs at the tops of files, like at https://dev.searchfox.org/mozilla-central/source/js/xpconnect/loader/XPCOMUtils.sys.mjs and if you click on them they offer, using an example of https://searchfox.org/mozilla-central/source/dom/workers/JSSettings.h [search which shows uses](https://searchfox.org/mozilla-central/search?q=symbol:FILE_dom%2Fworkers%2FJSSettings%402Eh&redirect=false), [calls-to diagram which shows the include dependency graph that ends up including the file](https://searchfox.org/mozilla-central/query/default?q=calls-to%3A%27dom%2Fworkers%2FJSSettings.h%27%20depth%3A4), and [calls-from which shows all the files that file includes and transitively too](https://searchfox.org/mozilla-central/query/default?q=calls-from%3A%27dom%2Fworkers%2FJSSettings.h%27%20depth%3A4).

I think this should automatically work for diagrams if you add the target records, but it is possible that I may have put some guards in place somewhere that inhibit things... if that happens it should be easy enough to fix and the immediate ability to search for the inclusions of the file I think would be useful.